ts/store/transforms/potTransforms.ts

```diff
@@ -0,0 +1,60 @@
+import { createTransform } from "redux-persist";
+
+import { resetPotStatus } from "../../utils/pot";
+import { GlobalState } from "../reducers/types";
+
+/**
+ * A redux-persist transformer that reset the status of the pots
+ */
+export const PotTransform = createTransform(
+  // We only care about rehydrated so we do not apply any transformation
+  // for inboundState.
+  _ => _,
+  (outboundState, key) => {
+    if (key === "entities") {
```

Comment:
I think we can assume that is safe to apply this transformation to any pot value as _loading_ and _error_ states should be reset on rehydration - what do you think about this?

Reply:
What you exactly mean with "apply this transformation to any pot value"? You mean you want to create others "if" for other sections of the store that use pot values?

Reply:
I mean doing something [like this](https://github.com/rt2zz/redux-persist/issues/82#issuecomment-468265789)

Reply:
Really a good solution! Let me check if it works.

Reply:
btw we should use that technique also for `Date`s to avoid having the `PO` models